controller/tests/products.spec.js

```diff
@@ -1,13 +1,62 @@
 const knex = require('knex');
 const knexConfig = require('../../knexfile');
 
-const db = knex(knexConfig.test);
+jest.mock('../../data/dbConfig');
 
+const db = require('../../data/dbConfig');
+const test_db = knex(knexConfig.test);
+
+db.mockImplementation((table) => {
+  return test_db(table);
+});
+
+const {
+  newProd,
+  getProds
+} = require('../products');
 
 describe('Model - Products:', () => {
+  beforeAll(async () => {
+    // Use test DB instead of dev
+    const test_db = knex(knexConfig.test);
+    // Migrate to our test DB in memory
+    await test_db.migrate.latest();
+    // Mock db call in file
+    db.mockImplementation((table) => {
+      return test_db(table);
+    });
```

Comment:
Uncaught promises are deprecated.
If you want to use async / await, you'll have to use the 
```
try {
} catch (err) {
}
```
Syntax

Reply:
You've also suggested something like this:

```
const x = await someFunction().catch(err=>whatever);
```

I find that a lot more elegant instead of doing try/catch for just one await statement, so I'm going to roll with that. Thanks for the feedback!